src/test/java/org/fcrepo/migration/f4clients/OCFLFedora4ClientIT.java

```diff
@@ -90,4 +91,15 @@ public void testCreateResource() {
         final String id = UUID.randomUUID().toString();
         client.createResource(id);
     }
+
+    @Test
+    public void testCreateOrUpdateNonRDFResource() {
+        final String path = UUID.randomUUID().toString();
+        client.createResource(path);
+        client.createOrUpdateNonRDFResource(path + "/v1/content/file.xml",
+                new ByteArrayInputStream("<sample>test-1</sample>".getBytes()), "text/xml");
+        client.createOrUpdateNonRDFResource(path + "/v1/content/file.xml",
+                new ByteArrayInputStream("<sample>test-2</sample>".getBytes()), "text/xml");
+        // TODO - check the content of the file?
```

Comment:
Is there a reason you are not checking that the update succeeded?

Reply:
I was relying on the exceptions thrown by the method. I'm not sure how to check the contents of the file without hard coding the path.